packages/cli/src/commands/setup/ui/libraries/mantine.js

```diff
@@ -0,0 +1,97 @@
+import execa from 'execa'
+import Listr from 'listr'
+
+import c from '../../../../lib/colors'
+import {
+  checkStorybookStatus,
+  configureStorybook,
+} from '../tasks/configure-storybook'
+import {
+  checkSetupStatus,
+  wrapWithMantineProvider,
+} from '../tasks/setup-mantine'
+
+export const command = 'mantine'
+export const description = 'Set up Mantine UI'
+
+const ALL_KEYWORD = 'all'
+const ALL_MANTINE_PACKAGES = [
```

Comment:
@reviewer Is there a more future-proof approach here where we can query NPM and get all of the published subpackages of mantine?

I do wonder if this is desirable; maybe we want to have a human being decide to add some future-published subpackage to this list. My instinct is that automatic is better, and we should only be delicate about the default value of `[core, hooks]` since I suspect that's what the vast majority of users will invoke.

Reply:
My preference is to manage this list manually. If there are new additions, we'll update the list and then also communicate the change in Release Notes so people can be reminded to add new packages.

That said, I don't know matine.